.env.sample

```diff
@@ -3,4 +3,9 @@ BASEURL_DISCOVERY="https://<base_url>/v2"  # Space and Time Discovery API Endpoi
 USERID="" # UserID required for authentication and authorization
 JOINCODE="" # Space and Time Join Code which can be got from the SxT release team
 SCHEME="ed25519"  # The key scheme or algorithm required for key generation
-PREFIX="" # Optional Prefix Parameter for signature generation
\ No newline at end of file
+PREFIX="" # Optional Prefix Parameter for signature generation
+
+# TEST
```

Comment:
Do you mean to add these?

Reply:
Right. This isn't required.

Reply:
No not the prefix - the test vars below this comment. I think you do actually need the PREFIX

Reply:
The test credentials are for running unit tests during the CI check with Github workflow and also, if one needs to run the tests locally

Reply:
Actually, I have checked for the Prefix and I haven't used the optional param anywhere